.gitlab/deploy_packages/nix.yml

```diff
@@ -126,6 +138,18 @@ deploy_packages_dogstatsd_rpm-x64-7:
   variables:
     PACKAGE_ARCH: x86_64
 
+deploy_packages_agentless_scanner_rpm-x64-7:
+  extends: .deploy_packages_rpm-7
+  needs: [ agentless_scanner_rpm-x64 ]
+  variables:
+    PACKAGE_ARCH: x86_64
+
+deploy_packages_agentless_scanner_rpm-arm64-7:
+  extends: .deploy_packages_rpm-7
+  needs: [ agentless_scanner_rpm-arm64 ]
+  variables:
+    PACKAGE_ARCH: arm64
```

Comment:
Isn't `aarch64` the correct arch for arm64 on RPM?

Reply:
The other rules use `arm64`.